cmd/calyptia/get_cluster_objects.go

```diff
@@ -0,0 +1,199 @@
+		return nil, cobra.ShellCompDirectiveError
+	}
+
+	if pp == nil {
+		return nil, cobra.ShellCompDirectiveNoFileComp
+	}
+
+	return clusterObjectsKeys(pp), cobra.ShellCompDirectiveNoFileComp
+}
+
+// ClusterObjectsKeys returns unique cluster object names first and then IDs.
+func clusterObjectsKeys(aa []cloud.ClusterObject) []string {
+	namesCount := map[string]int{}
+	for _, a := range aa {
+		if _, ok := namesCount[a.Name]; ok {
+			namesCount[a.Name] += 1
+			continue
+		}
+
+		namesCount[a.Name] = 1
+	}
+
+	var out []string
+
+	for _, a := range aa {
+		var nameIsUnique bool
+		for name, count := range namesCount {
+			if a.Name == name && count == 1 {
+				nameIsUnique = true
+				break
+			}
+		}
+		if nameIsUnique {
+			out = append(out, a.Name)
+			continue
+		}
+
+		out = append(out, a.ID)
+	}
+
+	return out
+}
+
+func (config *config) loadClusterObjectID(key string, environmentID string) (string, error) {
+	aa, err := config.fetchAllClusterObjects()
+	if err != nil {
+		return "", err
+	}
+
+	for _, a := range aa {
+		if a.Name == key {
+			return a.ID, nil
+		}
+	}
+
+	return key, nil
+}
+
+func newCmdGetClusterObjects(config *config) *cobra.Command {
+	var coreInstanceKey string
+	var last uint
+	var outputFormat, goTemplate string
+	var environment string
+	var showIDs bool
+
+	cmd := &cobra.Command{
+		Use:   "cluster_objects",
+		Short: "Get cluster objects",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			var environmentID string
+			if environment != "" {
+				var err error
+				environmentID, err = config.loadEnvironmentID(environment)
+				if err != nil {
+					return err
+				}
+			}
+
+			coreInstanceID, err := config.loadCoreInstanceID(coreInstanceKey, environmentID)
+			if err != nil {
+				return err
+			}
+
+			co, err := config.cloud.ClusterObjects(config.ctx, coreInstanceID, cloud.ClusterObjectParams{
+				Last: &last,
+			})
+			if err != nil {
+				return fmt.Errorf("could not fetch your cluster objects: %w", err)
+			}
+
+			if strings.HasPrefix(outputFormat, "go-template") {
+				return applyGoTemplate(cmd.OutOrStdout(), outputFormat, goTemplate, co.Items)
+			}
+
+			switch outputFormat {
+			case "table":
+				{
+					tw := tabwriter.NewWriter(cmd.OutOrStdout(), 0, 4, 1, ' ', 0)
+					if showIDs {
+						fmt.Fprintf(tw, "ID\t")
+					}
+					fmt.Fprintln(tw, "NAME\tKIND\tCREATED AT")
+					for _, c := range co.Items {
+						if showIDs {
+							fmt.Fprintf(tw, "%s\t", c.ID)
+						}
+						fmt.Fprintf(tw, "%s\t%s\t%s\n", c.Name, string(c.Kind), fmtTime(c.CreatedAt))
+					}
+					tw.Flush()
+				}
+			case "json":
+				return json.NewEncoder(cmd.OutOrStdout()).Encode(co.Items)
+			case "yml", "yaml":
+				return yaml.NewEncoder(cmd.OutOrStdout()).Encode(co.Items)
+			default:
+				return fmt.Errorf("unknown output format %q", outputFormat)
+			}
+			return nil
+		},
+	}
+
+	fs := cmd.Flags()
+	fs.StringVar(&coreInstanceKey, "core_instance", "", "Core Instance to list cluster objects from")
```

Comment:
```suggestion
	fs.StringVar(&coreInstanceKey, "core-instance", "", "Core Instance to list cluster objects from")
```

For the most part, we have been using dash-case for the flags.